lib/racecar/config.rb

```diff
@@ -34,81 +43,18 @@ class Config < KingKonf::Config
     desc "Whether to exponentially increase the pause timeout on successive errors -- the timeout is doubled each time"
     boolean :pause_with_exponential_backoff, default: false
 
-    desc "The idle timeout after which a consumer is kicked out of the group"
-    float :session_timeout, default: 30
-
-    desc "How long to wait when trying to connect to a Kafka broker"
-    float :connect_timeout, default: 10
-
-    desc "How long to wait when trying to communicate with a Kafka broker"
-    float :socket_timeout, default: 30
-
-    desc "How long to allow the Kafka brokers to wait before returning messages"
-    float :max_wait_time, default: 1
-
-    desc "The maximum size of message sets returned from a single fetch"
-    integer :max_bytes, default: 10485760
```

Comment:
So of these are available in rdkafka – I'd like to keep these high-level config keys and just copy to the matching ones in rdkafka.